Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).


It also includes following components:

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/keycloak/admin-ui/horizontal-auto.svg)

ahus1 commented on themes/src/main/resources-community/theme/base/admin/messages/messages_fr.properties:
```diff
@@ -7,3 +7,12 @@ invalidPasswordNotUsernameMessage=Mot de passe invalide : ne doit pas être iden
+invalidPasswordNotEmailMessage=Mot de passe invalide : il ne doit pas correspondre à l'adresse de courriel.
+invalidPasswordBlacklistedMessage=Mot de passe invalide : mot de passe blacklisté.
+invalidPasswordGenericMessage=Mot de passe invalide : le nouveau mot de passe ne respecte pas les politiques de mot de passe.
+ldapErrorEditModeMandatory=Le mode d'édition est obligatoire
```
Thank you for this translation, @Dodouce!

Once thing to note is that for all backend messages, all single quotes need to be duplicated, while for the frontend they must not be. 

See [the docs](https://github.com/keycloak/keycloak/blob/main/docs/translation.md#handling-of-single-quotes-)  for a more detailed explanation where to use what. 

The automatic check flagged those items, and I will go ahead and fix them in Weblate. For future translations, please try to use the correct double single quotes.

Alternatively, there are typographic quote that will work just normal in the frontend and the backend. Maybe 
 the two language maintainers @jmorille and @blackholegalaxy have an opinion here as well.

weblate committed:
Updated translation for Portuguese (Brazil)

Language: pt_BR

Updated translation for Portuguese (Brazil)

Language: pt_BR

Updated translation for Portuguese (Brazil)

Language: pt_BR

Translated using Weblate (Portuguese (Brazil))

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/pt_BR/

Updated translation for Portuguese (Brazil)

Language: pt_BR

Updated translation for Portuguese (Brazil)

Language: pt_BR

Updated translation for Portuguese (Brazil)

Language: pt_BR

Updated translation for Portuguese (Brazil)

Language: pt_BR

Updated translation for Portuguese (Brazil)

Language: pt_BR

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Lucas Pugliese <lucas.pugliese.barros@gmail.com>
Co-authored-by: Rafael Cunha <rafael.danicunha@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Lucas Pugliese <lucas.pugliese.barros@gmail.com>
Signed-off-by: Rafael Cunha <rafael.danicunha@gmail.com>

weblate committed:
Updated translation for Romanian

Language: ro

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Razvan Tufisi <rtufisi@phasetwo.io>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Razvan Tufisi <rtufisi@phasetwo.io>

weblate committed:
Updated translation for Japanese

Language: ja

Updated translation for Japanese

Language: ja

Updated translation for Japanese

Language: ja

Updated translation for Japanese

Language: ja

Updated translation for Japanese

Language: ja

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kohei Tamura <ktamura.biz.80@gmail.com>
Co-authored-by: Takashi Norimatsu <takashi.norimatsu.ws@hitachi.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Kohei Tamura <ktamura.biz.80@gmail.com>
Signed-off-by: Takashi Norimatsu <takashi.norimatsu.ws@hitachi.com>

weblate committed:
Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Translated using Weblate (French)

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/fr/

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Translated using Weblate (French)

Translation: Keycloak/Theme base/admin
Translate-URL: https://hosted.weblate.org/projects/keycloak/theme-baseadmin/fr/

Updated translation for French

Language: fr

Updated translation for French

Language: fr

Co-authored-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Co-authored-by: Dodouce <marin.pau22@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Sylvain Pichon <service@spichon.fr>
Signed-off-by: Alexander Schwartz <alexander.schwartz@gmx.net>
Signed-off-by: Dodouce <marin.pau22@gmail.com>
Signed-off-by: Hosted Weblate <hosted@weblate.org>
Signed-off-by: Sylvain Pichon <service@spichon.fr>

ahus1 commented:
@liviuroman / @edwint88: The Romanian changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@y-tabata / @wadahiro / @tnomrimat: The Japanese changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@felipebz / @rafaelrddc / @julianorodrigox: The Portuguese (Brazil) changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.

@jmorille / @blackholegalaxy: The French changes are now merged to Keycloak's main branch. Please review the changes in this PR, and add corrections to Weblate as necessary.
